apps/backoffice-v2/src/common/components/organisms/Filters/Filters.Header.tsx

```diff
@@ -0,0 +1,10 @@
+import { IFiltersHeaderProps } from '@/common/components/organisms/Filters/interfaces';
+import { FunctionComponent } from 'react';
+
+export const FiltersHeader: FunctionComponent<IFiltersHeaderProps> = ({ title }) => {
+  return typeof title === 'string' ? (
+    <h4 className={'leading-0 min-h-[16px] pb-7 text-xs font-bold'}>{title}</h4>
+  ) : (
+    title
+  );
```

Comment:
Can you explain this? Why was this needed, etc.

Reply:
In case if custom styles or `React.Node` will be needed.

Reply:
I think we will get considerably  more value by using something like `as` or `asChild` or just `children` and passing `className` into `ctw`. Changing the markup depending on string to me is unpredictable. Otherwise if we make a re-usable filters component the heading will probably be a part of it and won't be elsewhere.